docker/suite/Dockerfile

```diff
@@ -9,8 +9,9 @@ RUN apt-get update && apt-get -y --no-install-recommends install \
     ca-certificates \
     curl
 
-RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.4/gosu-$(dpkg --print-architecture)" \
-    && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.4/gosu-$(dpkg --print-architecture).asc" \
+# TODO: for M1 doesn't work
+RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-$(dpkg --print-architecture)" \
+    && curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-$(dpkg --print-architecture).asc" \
     && chmod +x /usr/local/bin/gosu
```

Comment:
I am not even 100% sure we need that line. I am wondering what happens if you remove it altogether. The `docker/suite/Dockerfile` needs some love and will be addressed.

Reply:
Without it, running the image fails for unsufficient permissions. (I'd be sruprised if curl would be able to spit out executable file.)

> The docker/suite/Dockerfile

Agree. For example, it's amost a perfect duplicate of `docker/test-runner/Dockerfile` modulo the base image. I consider that out of scope. (This PR already being quite far from the scope from the feature I'm [working on now](https://github.com/trezor/trezor-suite/pull/3590)).

Reply:
This PR is good from my side. If you're happy, @tsusanka, please resolve this thread and merge. (I'm not authorized to do that.)